easybuild/framework/easyblock.py

```diff
@@ -1523,6 +1524,12 @@ def prepare_step(self):
         # prepare toolchain: load toolchain module and dependencies, set up build environment
         self.toolchain.prepare(self.cfg['onlytcmod'], silent=self.silent)
 
+        self.log.debug("prepare_step: PATH %s" % os.environ['PATH'])
+        if build_option('rpath'):
+            # Setup the environment and copy wrapper script into path
+            prepare_ld_wrapper()
```

Comment:
hmm, shouldn't we do this in the `toolchain.prepare()` itself? keep things nicely together?